subprojects/gstreamer/gst/gstvalue.c

```diff
@@ -8052,11 +8052,14 @@ gst_value_transform_allocation_params_string (const GValue * value1,
   GstStructure *s;
 
   s = gst_structure_new_id (GST_QUARK (ALLOCATION_PARAMS),
-      GST_QUARK (FLAGS), GST_TYPE_MEMORY_FLAGS, params->flags,
       GST_QUARK (ALIGN), G_TYPE_UINT64, params->align,
       GST_QUARK (PREFIX), G_TYPE_UINT64, params->prefix,
-      GST_QUARK (PADDING), G_TYPE_UINT64, params->padding, NULL);
-
+      GST_QUARK (PADDING), G_TYPE_UINT64, params->padding,
+      /* Keeping GST_TYPE_MEMORY_FLAGS as the first va_arg makes a function
```

Comment:
You can use `#ifdef (__EMSCRIPTEN__)`...

Reply:
I would like to keep this as is. I was investigating the memory corruption but couldn't find it. This needs to be fixed properly, but it is okay for now. The community will reject this one too.